src/app/settings/preferences.component.ts

```diff
@@ -82,6 +82,15 @@ export class PreferencesComponent implements OnInit {
   }
 
   async submit() {
+    if (this.vaultTimeout.errors?.minTimeoutError) {
```

Comment:
I know Danielle asked for this, but I don't want a pattern where we have to handle individual errors at the form level. It just doesn't scale, we'll end up with a million `if` statements. Can you check with her whether we can just change this to a generic message, like "The form has errors, please review and try again"? (or whatever wording she suggests)